make_env.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+echo export OPENSHIFT_CFME_ROUTE=\""$(oc get route --namespace='openshift-management' -o go-template --template='{{.spec.host}}' httpd 2> /dev/null)"\"
+echo export OPENSHIFT_HAWKULAR_ROUTE=\""$(oc get route --namespace='openshift-infra' -o go-template --template='{{.spec.host}}' hawkular-metrics 2> /dev/null)"\"
+echo export OPENSHIFT_PROMETHEUS_ALERTS_ROUTE=\""$(oc get route --namespace='openshift-metrics' -o go-template --template='{{.spec.host}}' alerts 2> /dev/null)"\"
+echo export OPENSHIFT_PROMETHEUS_METRICS_ROUTE=\""$(oc get route --namespace='openshift-metrics' -o go-template --template='{{.spec.host}}' prometheus 2> /dev/null)"\"
+echo export OPENSHIFT_MASTER_HOST=\""$(oc get nodes -o name |grep master |sed -e 's/nodes\///g')"\"
+echo export OPENSHIFT_MANAGEMENT_ADMIN_TOKEN=\""$(oc sa get-token -n management-infra management-admin)"\"
+echo export OPENSHIFT_CA_CRT=\""$(cat /etc/origin/master/ca.crt)"\"
+echo export OPENSHIFT_PROVIDER_NAME=\""OCP37"\"
+echo export OPENSHIFT_CFME_USER=\""admin"\"
+echo export OPENSHIFT_CFME_PASS=\""smartvm"\"
```

Comment:
BTW There is an ability to specify a password when creating CFME template.
We would need to use that at some point (as ops are going to)